packages/ecs-agent/ecs.config

```diff
@@ -16,3 +16,9 @@ ECS_ENGINE_AUTH_DATA='{
         {{~#if password~}},"password": "{{{password}}}"}{{/if}}
     {{~/each~}}}}'
 {{/if}}
+{{#if settings.ecs.container-stop-timeout}}
+ECS_CONTAINER_STOP_TIMEOUT="{{settings.ecs.container-stop-timeout}}"
+{{/if}}
+{{#if settings.ecs.task-cleanup-wait-duration}}
+ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION="{{settings.ecs.task-cleanup-wait-duration}}"
```

Comment:
nit: I think `task-stop-timeout` and `task-cleanup-wait` would be more consistent, naming-wise.

(dropping "duration" from `task-cleanup-wait` seems valuable since it's sort of implied that it's a Duration)

Reply:
I think `task-stop-timeout` could be inaccurate, since this value is used for when stopping a container in a task (and there could be plenty of them) not when stopping a task.